### PR TITLE
Added GlusterFS storage optimization content for 4.0

### PR DESCRIPTION
**Pull request by ahardin-rh**
Preview build: http://file.rdu.redhat.com/~ahardin/01072019/glusterfs-optimization-4-0/scalability_and_performance/glusterfs-storage-optimization.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jeremyeder PTAL

**Comment by jeremyeder** [comment]
This is @ekuric document

File: modules/tested-applications-for-glusterfs-storage.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//
+// scalability_and_performance/glusterfs-storage-optimization.doc
+
+[id='tested-applications-for-glusterfs-storage-{context}']
+= Tested applications for GlusterFS storage
+
+Red Hat performed extensive testing with these (no)SQL databases:
+
+-  Postgresql SQL v9.6
+-  MongoDB noSQL v3.2
```
Comment (ekuric) [inline modules/tested-applications-for-glusterfs-storage.adoc]: I need to check what version of Mongodb/Postgresql will be with OCP v4.0 templates

**Comment by ekuric** [comment]
@ahardin-rh this is to my best knowledge be true in ocp v4.0 when it comes to glusterfs,but let me check versions and will follow up

**Comment by ahardin-rh** [comment]
@ekuric @jeremyeder Closing this per earlier discussion

**Pull request closed without merging**